ROS/controls/src/nuc_motor_controls.cpp

```diff
@@ -0,0 +1,119 @@
+#include "nuc_motor_controls.h"
```

Comment:
Is this the most recent version (and finished) of david's file? If not, please unadd it from the commit. If it is let me know by replying to this comment

Reply:
It's the most recent but some calculation changes might need to be made. We can put another pull request for later with the changes but for now I think we should add it so that it's there at least. Furthermore a lot would need to be removed from the commit (the CMake, launch file, etc)

Reply:
I'll add some changes in the next commit (which will fix the other issues too) which will have better calculations in a little while

Reply:
Ok sounds good